src/ducks/modules/rootReducer.js

```diff
@@ -10,6 +10,7 @@ import draggable from './draggable';
 import droppable from './droppable';
 import modals from './modals';
 import menu from './menu';
+import search from './search';
```

Comment:
Is this module defunct now?

Reply:
It is being used, though arguably we don't really need to keep the toggle state of the search panel.